### PR TITLE
Game spec

### PR DESCRIPTION
**Pull request by Adewunmi97**
-Added games_spec file
-Added RSpec test cases for different methods

### CODE REVIEW

**Review by javitocor — CHANGES_REQUESTED** [review]
# STATUS CHANGES REQUIRED ♻️ 

Hello @Lidya1234 and @Adewunmi97!

Good job so far with this project! 👏 
But we have to make some changes before getting approved! 💪 
Kindly check my comments below 👇  apply the changes and resubmit your project for a new code review
Good luck! 😃 

## Mandatory changes
- Your linters are failing, kindly make them all pass
- One of your tests is failing, kindly make it pass.
   - RSpec ./spec/games_spec.rb:32 # game method tests Validates position
- Add test section in your readme file
- You must do tests for all your public methods, there are still methods without a test, kindly add those tests

File: README.md [hunk 1]
```diff
@@ -37,19 +40,20 @@ This is a project to build a Tic tac toe game(a 3X3 board_game) using ruby and p
...
-9.If a player filled consecutive row, columns, or diagonal it announces the winner 
+9.If a player filled consecutive row, columns, or diagonal it announces the winner
 
 10.If it does not find a winner it is going to display Game is a draw message
 
 ## Game Instructions
 Tic-tac-toe is a board game for two players, X and O, who take turns marking the spaces in a 3×3 grid. The player who succeeds in placing three of their marks in a horizontal, vertical, or diagonal row is the winner. It is a solved game with a forced draw assuming best play from both players.
 
 <img src="game_rules.png">
+<img src="Screenshot.png">
```
Comment (moinkhanif) [inline README.md:54]: [OPTIONAL] Add test screenshot within a test heading.

File: bin/main [hunk 2]
```diff
@@ -0,0 +1,39 @@
+
```
Comment (moinkhanif) [inline bin/main:1]: [OPTIONAL]  But important, rename main to main.rb

**Review by moinkhanif — APPROVED** [review]
##  Status: Approved! 🥳  

Hey team, great work on your project ✨. You have completed the requested changes by the previous TSE. Therefore **I am approving** your project. 🎉
However, please do checkout the optional changes I have requested below:
- Please add negative tests as well. Having both positive and negative tests makes sure the project is running as expected.
- The rest are commented below.

You may now merge the branch. Hope you enjoy upcoming projects and understand the concepts better! 😄

Happy coding! 🐱‍🏍

**Pull request merged**